test/routes/article-list.test.ts

```diff
@@ -23,14 +23,35 @@ describe('article list', (): void => {
     expect(response.status).toBe(OK);
   });
 
-  it('should return an empty list', async (): Promise<void> => {
+  it('should return a collection', async (): Promise<void> => {
+    const { dataset } = await makeRequest();
+    const id = namedNode('http://example.com/path-to/article-list');
+
+    expect(dataset).toBeRdfDatasetContaining(
+      quad(id, rdf.type, hydra.Collection),
+      quad(id, hydra.title, literal('List of articles', 'en')),
+    );
+  });
+
+  it('should contain articles', async (): Promise<void> => {
     const { dataset } = await makeRequest();
     const id = namedNode('http://example.com/path-to/article-list');
 
-    expect(dataset).toBeRdfDatasetContaining(quad(id, rdf.type, hydra.Collection));
-    expect(dataset).toBeRdfDatasetMatching({ subject: id, predicate: hydra.title });
     expect(dataset).toBeRdfDatasetMatching({ subject: id, predicate: hydra.manages });
-    expect(dataset).toBeRdfDatasetMatching({ subject: id, predicate: hydra.totalItems, object: toRdf(0) });
+
+    const [{ object: manages }] = dataset.match(id, hydra.manages);
```

Comment:
if the predicate is `manages` (namespaces notwithstanding), is the object `manages` too? Maybe `managed`?

Reply:
or just `article`?

Reply:
assume if 2 articles suddenly come out the API, the `[{...}]` destructuring fails?

Reply:
oh wait, this only describes the type of the items, not their content

Reply:
It's assigning an alias to a property on an object that is the first item in the array. Complicated syntax...

Destructuring is fine if there's more than one, others are ignored.